randovania/layout/game_patches_serializer.py

```diff
@@ -175,6 +224,7 @@ def decode_single(
     starting_location = NodeIdentifier.from_string(game_modifications["starting_location"])
 
     # Starting Equipment
+    starting_equipment: list | ResourceCollection
```

Comment:
shouldn't this be `game_patches.StartingEquipment`?

Reply:
I forgot that alias exist. Good catch